plugins/obs-outputs/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ project(obs-outputs)
 set(WITH_RTMPS AUTO CACHE STRING "Enable RTMPS support with mbedTLS")
 set_property(CACHE WITH_RTMPS PROPERTY STRINGS AUTO ON OFF)
 
+find_package(PkgConfig)
```

Comment:
Require and remove the if PKG_CONFIG_FOUND check.

Reply:
Woops wasnt thinking of windows... disregard.

Reply:
I don't think pkg-config should be a hard requirement just to "maybe" have the possibility of using one system libraries instead of the in-tree one.